lib/mongo/crypt/auto_encrypter.rb

```diff
@@ -160,7 +160,7 @@ def spawn_mongocryptd
       #
       # @return [ true ] Always true
       def teardown_encrypter
-        @mongocryptd_client.close if @mongocryptd_client
+        #@mongocryptd_client.close if @mongocryptd_client
```

Comment:
This should be investigated also.

Reply:
What exactly needs to be investigated here?

Reply:
If the comment is removed, is the CI still green?

Reply:
Yup!